Assignment/solution_student/index.html

```diff
@@ -1,35 +1,66 @@
+<!DOCTYPE HTML>
 
-Sample Content
-——-
+<html>
+<head>
+	<title>Home</title>
+	<meta <meta charset="utf-8">
+	<link rel="stylesheet" href="css/main.css">
+</head>
+<h5 id="zz"><strong>About me</strong> | <a href="#">Portfolio</a> | <a href="resume.html">Resume</a></h5>
```

Comment:
Oops! Looks like you forgot an important element here...it comes directly after the `<head>` and it contains aaaaall of the content of the web page...